tests/test-conf.yaml

```diff
@@ -1,3 +1,20 @@
-# I propose to remove this file in favor of a fixture with similar name
-# fixture can be constructed and written more easily, without current dir joining
-# also it can be imported elsewhere more easily
+dispatcher:
```

Comment:
I'd put a note here which I put, this one:

```
# I propose to remove this file in favor of a fixture with similar name
# fixture can be constructed and written more easily, without current dir joining
# also it can be imported elsewhere more easily
```

or remove  the file entirely

Reply:
> I'd put a note here which I put, this one:
> 
> ```
> # I propose to remove this file in favor of a fixture with similar name
> # fixture can be constructed and written more easily, without current dir joining
> # also it can be imported elsewhere more easily
> ```
> 
> or remove the file entirely

I will remove the file entirely. With the next commit